tests/unit/miniapps/test_debug_device.cpp

```diff
@@ -87,6 +88,15 @@ void Aliases(const int N = 0x1234)
    REQUIRE(S*S == MFEM_Approx(24.0*N));
 }
 
+TEST_CASE("Array::MakeRef", "[DebugDevice]")
+{
+   Array<int> x(1);
+   Array<int> y;
+   y.MakeRef(x);
+   x.Read();
+   REQUIRE_NOTHROW(y.Read());
```

Comment:
I'm not sure we want to enforce this: after `x.Read()`, `y` is out of sync, so requiring that `y.Read()` works without a sync is a strong requirement.

@camierjs, what do you think?

Reply:
I'm fine leaving it there as it is not throwing the memory error.

Reply:
Okay, let's keep it as it is.